components/match2/wikis/halo/match_summary.lua

```diff
@@ -144,14 +144,32 @@ function CustomMatchSummary._createMapRow(game)
 		centerNode:addClass('brkts-popup-spaced-map-skip')
 	end
 
+	local getScoreValue = function(opponentIndex)
+		local score = game.scores[opponentIndex]
+		if not score then
+			return
+		end
+		local points = game.extradata['points' .. opponentIndex]
+		if not points then
+			return score
+		end
+		return score .. ' (' .. points .. ')'
```

Comment:
@Kanopedia let me know whether you want flipping as well
```suggestion
        if opponentIndex == 1 then
            return '(' .. points .. ') ' .. score
        end
		return score .. ' (' .. points .. ')'
```

Reply:
This was the flip mentioned on discord so that the scoring is inwards on both sides?

Reply:
![image](https://github.com/user-attachments/assets/14a3e962-194f-48a6-b4df-b60c533f1b4e)